scripts/launch/node.sh

```diff
@@ -21,6 +21,8 @@ do
         node="witness"
     elif [[ "$var" == "educator" ]] || [ "$var" == "e" ]; then
         node="educator"
+    elif [[ "$var" == "educator" ]] || [ "$var" == "me" ]; then
```

Comment:
So the first part of this OR will never take place anyway (`educator` case is matched already 2 lines above), maybe better provide `multi-educator`?

Reply:
Please, don't copy-paste lines.